php/libraries/Utility.class.inc

```diff
@@ -539,7 +539,7 @@ class Utility
      */
     static function toArray($var)
     {
-        if (!isset($var[0])) {
+        if (!array_key_exists(0, $var)) {
```

Comment:
get a the following php warning when passing a string to function
`
PHP Warning:  array_key_exists() expects parameter 2 to be array
`